modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -233,10 +233,29 @@ private void handleUpdateCommand(UpdateCommand cmd, long commandIndex, long comm
 
         storageUpdateHandler.handleUpdate(cmd.txId(), cmd.rowUuid(), cmd.tablePartitionId().asTablePartitionId(), cmd.rowBuffer(),
                 rowId -> {
-                    txsPendingRowIds.computeIfAbsent(cmd.txId(), entry -> new TreeSet<>()).add(rowId);
+                    if (!cmd.full()) {
+                        txsPendingRowIds.computeIfAbsent(cmd.txId(), entry -> new TreeSet<>()).add(rowId);
+                    } else {
+                        TxMeta txMetaToSet = new TxMeta(
+                                COMMITED,
```

Comment:
Because one-phase tx is the same tx as others. For example, the primary replica is updated, but replication command has failed. Such transaction requires cleanup.

Reply:
I've removed accessing the state for one-phase txns because we can live without it. It's not clear at the moment if we will need this state for one-phase txns later.